### ISSUE TITLE
Widget item font size different from app setting

### ISSUE THREAD

**Issue by xanharris** [mpcjanssen/simpletask-android#275]
When using v6.0.3, I found that I'm seeing less items in the widget. Prior versions (Before Material updates, I think) did not have this problem as I can see 5 items. Now, I can only see 3.5 items. The font size should follow the size specified in the app setting.

Thank you and keep up the good work!

**Comment by mpcjanssen** [comment]
Do you use the Small font setting?

**Comment by xanharris** [comment]
No, as I find it too small :-) Am using Medium, but the widget is using a larger font. Since you mentioned it, I tried small, but no effect on the widget.

FYI, am using HTC One M7 running KitKat 4.4.2.

**Comment by mpcjanssen** [comment]
Because the application wide theme changed to Material this could be the
cause. I can see if I can add a widget specific font size setting.

**Comment by smichel17** [comment]
If possible, I'd encourage having the widget inherit font size from the main app rather than adding another setting.

**Comment by smichel17** [comment]
duplicate of #166?

**Comment by mpcjanssen** [comment]
@smichel17 similar yes. It's all different aspects of configuring the UI. I would like to add a completely separate section where one can configure all UI aspects instead of fixing it one by one.

**Comment by smichel17** [comment]
Related: #412

**Comment by Henry78** [comment]
Is the widget font size configurable through lua? How?
